wazo_sysconfd/plugins/systemd_services/services.py

```diff
@@ -66,12 +51,13 @@ def _run_action_for_service_validated(service, action):
         logger.debug("%s : return code %d", ' '.join(command), p.returncode)
 
         if p.returncode != 0:
-            raise HttpReqError(500, output)
+            raise exceptions.InternalServerErrorException(500, output)
```

Comment:
Changing the exception type will most likely break the error handling here. Please keep the original type.

Reply:
Reverted